README.md

```diff
@@ -29,7 +29,7 @@ edge computing, multiparty computing (including deterministic machine learning),
 client-side-validation, sandboxed Internet2 computing and genetic algorithms.
 
 For more details on AluVM, please check [the specification][AluVM], watch
-detailed presentation on [YouTube] or [check slides] from the presentation.
+detailed presentation on [YouTube](https://www.youtube.com/watch?v=brfWta7XXFQ) or check [slides](https://github.com/LNP-BP/presentations/blob/master/Presentation%20slides/RGB%20VM%20%26%20scripting.pdf) from the presentation.
```

Comment:
Look: markdown allows you to create hyperlinks in two ways: by using `[name](link)` notation or by just inserting `[name]` in the text and providing `[name]: link` and the end of the document. The latter allows you not to re-use the same link multiple times within the text itself.

I was using the latter notation, so there is no need to add `(link)` after each `[name]` in the text; the only problem that was there is the need to change `[check slides]` to either `check [slides]` or `[check slides][slides]`

Reply:
done